Client/Frontend/Menu/HomeMenuController.swift

```diff
@@ -176,11 +175,7 @@ class HomeMenuController: UIViewController, PopoverContentComponent {
     //switch to bookmarks 'tab' in case we're looking at history and tapped the add/remove bookmark button
     onClickPageButton(bookmarksButton)
     
-    if Bookmark.contains(url: url) {
-      Bookmark.remove(forUrl: url)
-    } else {
-      Bookmark.add(url: url, title: tabState.title, parentFolder: bookmarksController.currentBookmarksPanel().currentFolder)
-    }
+    Bookmark.add(url: url, title: tabState.title, parentFolder: bookmarksController.currentBookmarksPanel().currentFolder)
```

Comment:
I don't think this is still correct, the toggle should function as normal, it is just that the models should allow duplicate bookmarks to come from sync.

Or if the user edits two bookmarks to the same URL, that should be allowed (which I believe it currently is), but if the user hits the toggle while having a matching url bookmark it should be removed.

Other platforms currently delete a random bookmark if there are > 1 that match the given URL, and we have been told to follow suite on this too.

Reply:
So no code changes are needed. Removing bookmark by urls does just this, removes a random bookmark of a given url. 
This PR can be closed then